CommandHelper/LocalPackages/commands/buffs/zeus.ms

```diff
@@ -9,6 +9,24 @@
             'trail': '',
             'op': 'true',
             'speech': "Zeus Almighty.",
-            'powers': array(),
+            'powers': array(
+                "/buff",
+                "/lightning",
+                "/gmc",
+                "/fly",
+                "/kill,"
```

Comment:
Scales of Zeus Weighs soul to determine who dies or not in actuality Zeus controls the scales